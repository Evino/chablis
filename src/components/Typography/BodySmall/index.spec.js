import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import BodySmall from './';

const text = 'This is a text';
const element = <BodySmall>{text}</BodySmall>;
const component = mount(element);

describe('BodyLarge', () => {
  it('should render a text', () => {
    const tree = renderer
      .create(element)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should display passed text', () => {
    expect(component.prop('children')).toBe(text);
  });
});
