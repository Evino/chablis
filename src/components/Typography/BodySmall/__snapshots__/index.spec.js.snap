// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BodyLarge should render a text 1`] = `
<p
  className="BodySmall__ParagraphComponent-qab5dc-0 kkJvcZ"
>
  This is a text
</p>
`;
